--- wire/peer_exp_wire.csv	2021-10-18 11:43:51.851963867 -0500
+++ -	2021-10-18 11:43:56.623816578 -0500
@@ -127,6 +85,18 @@
 msgtype,stfu,2
 msgdata,stfu,channel_id,channel_id,
 msgdata,stfu,initiator,u8,
+msgtype,splice,74
+msgdata,splice,chain_hash,chain_hash,
+msgdata,splice,channel_id,channel_id,
+msgdata,splice,funding_feerate_perkw,u32,
+msgdata,splice,funding_pubkey,point,
+msgtype,splice_ack,76
+msgdata,splice_ack,chain_hash,chain_hash,
+msgdata,splice_ack,channel_id,channel_id,
+msgdata,splice_ack,funding_pubkey,point,
+msgtype,splice_locked,78,
+msgdata,splice_locked,channel_id,channel_id,
+msgdata,splice_locked,next_per_commitment_point,point,
 msgtype,shutdown,38
 msgdata,shutdown,channel_id,channel_id,
 msgdata,shutdown,len,u16,
@@ -164,6 +128,13 @@
 msgdata,commitment_signed,signature,signature,
 msgdata,commitment_signed,num_htlcs,u16,
 msgdata,commitment_signed,htlc_signature,signature,num_htlcs
+msgdata,commitment_signed,tlvs,commitment_signed_tlvs,
+tlvtype,commitment_signed_tlvs,splice_commitsigs,0
+tlvdata,commitment_signed_tlvs,splice_commitsigs,sigs,commitsigs,...
+subtype,commitsigs
+subtypedata,commitsigs,commit_signature,signature,
+subtypedata,commitsigs,num_htlcs,u16,
+subtypedata,commitsigs,htlc_signature,signature,num_htlcs
 msgtype,revoke_and_ack,133
 msgdata,revoke_and_ack,channel_id,channel_id,
 msgdata,revoke_and_ack,per_commitment_secret,byte,32
